layouts/partials/head.html

```diff
@@ -70,14 +76,36 @@
       // clear the graph in case there is anything within it
       container.textContent = ""
 
-      drawGraph(
-        {{strings.TrimRight "/" .Site.BaseURL}},
-        {{$.Site.Data.graphConfig.paths}},
-        {{$.Site.Data.graphConfig.depth}},
-        {{$.Site.Data.graphConfig.enableDrag}},
-        {{$.Site.Data.graphConfig.enableLegend}},
-        {{$.Site.Data.graphConfig.enableZoom}}
-      );
+      if (isHome && {{$.Site.Data.graphConfig.enableGlobalGraph}}) {
+        drawGraph(
+          {{strings.TrimRight "/" .Site.BaseURL}},
+          {{$.Site.Data.graphConfig.paths}},
+          {{$.Site.Data.graphConfig.depthGG}},
+          {{$.Site.Data.graphConfig.enableDragGG}},
+          {{$.Site.Data.graphConfig.enableLegendGG}},
+          {{$.Site.Data.graphConfig.enableZoomGG}},
+          true,
+          {{$.Site.Data.graphConfig.opacityScaleGG}},
+          {{$.Site.Data.graphConfig.scaleGG}},
+          {{$.Site.Data.graphConfig.repelForceGG}},
+          {{$.Site.Data.graphConfig.fontSizeGG}}
```

Comment:
can we change this to pass `{{$.Site.Data.graphConfig}}` instead? It'll just be a JavaScript object with all of these properties